node/src/server-modules/GlideJson.ts

```diff
@@ -661,4 +661,47 @@ export class GlideJson {
 
         return _executeCommand<ReturnTypeJson<number>>(client, args);
     }
+
```

Comment:
Could you tell what is **[-1]** in this example:
```
127.0.0.1:6379> JSON.SET k1 . '[[], ["a"], ["a", "b"]]'
OK
127.0.0.1:6379> JSON.ARRAPPEND  k1 [-1] '"c"'
(integer) 3
127.0.0.1:6379> JSON.GET k1
"[[],[\"a\"],[\"a\",\"b\",\"c\"]]"

``` 
Reference: https://docs.aws.amazon.com/AmazonElastiCache/latest/dg/json-arrappend.html

Reply:
`[-1]` means "last element in the array at doc root", so you commands adds `"c"` to `["a", "b"]`